app/(gcforms)/[locale]/(support)/support/components/client/SupportForm.tsx

```diff
@@ -69,6 +69,12 @@ export const SupportForm = () => {
         </p>
       </Alert.Warning>
       <form id="support" action={formAction} noValidate>
+        {state.error && (
+          <Alert.Danger focussable={true} title={t("error")} className="mb-2 mt-2">
+            <p>{t(state.error)}</p>
```

Comment:
We'll probably want to update the error content, same as https://github.com/cds-snc/platform-forms-client/pull/3689/.

This could be in another PR? In which case I can approve this PR?